mindmeld/query_cache.py

```diff
@@ -126,6 +131,24 @@ def compatible_version(self):
         except Exception:  # pylint: disable=broad-except
             return False
 
+    def compatible_text_pipeline_config(self):
+        """
+        Checks to see if the cache db file exists and that the text preparation pipeline config
+        matches the current pipeline config.
+        """
+
+        cursor = self.disk_connection.cursor()
+        try:
+            cursor.execute("""
```

Comment:
If you're only counting, you don't want to fetch an arbitrary number of rows. You can also simplify the condition below this.

```python
row = cursor.execute("""
    SELECT COUNT(version_number) FROM version WHERE text_pipeline_hash=(?)
""", (self.text_pipeline_hash,)).fetchone()
return row[0] > 0
```

Reply:
Updated!